src/libraries/Native/Unix/System.Native/pal_log.c

```diff
@@ -0,0 +1,12 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#include "pal_config.h"
+#include "pal_log.h"
+
+#include <stdio.h>
+
+void SystemNative_Log(uint8_t* buffer, int32_t length)
+{
+    fwrite(buffer, 1, (size_t)length, stdout);
```

Comment:
Just a note (that shoudl become a GH Issue) for Andriod and ios we will need to redirect the output similar to how `eglib/goutput.c` does it for `g_log(...)` https://github.com/dotnet/runtime/blob/8cb351833ab88788e49f6017d7c7ab3823fda5be/src/mono/mono/eglib/goutput.c#L237-L363